pkg/xds/bootstrap/testdata/generator.custom-config-minimal-request-and-delta.golden.yaml

```diff
@@ -0,0 +1,145 @@
+admin:
+  accessLog:
+  - name: envoy.access_loggers.file
+    typedConfig:
+      '@type': type.googleapis.com/envoy.extensions.access_loggers.file.v3.FileAccessLog
+      path: /var/log
+  address:
+    socketAddress:
+      address: 192.168.0.1
+      portValue: 9902
+defaultRegexEngine:
+  name: envoy.regex_engines.google_re2
+  typedConfig:
+    '@type': type.googleapis.com/envoy.extensions.regex_engines.v3.GoogleRE2
+dynamicResources:
+  adsConfig:
+    apiType: GRPC
```

Comment:
why is it GRPC and not GRPC_DELTA? (I assume this should be delta since the file is called delta)